spec/grape/validations/validators/default_spec.rb

```diff
@@ -63,6 +63,30 @@ class API < Grape::API
         get '/array' do
           { array: params[:array] }
         end
+
+        params do
+          requires :thing1
+          optional :more_things, type: Array do
+            requires :nested_thing
+            requires :other_thing, default: 1
```

Comment:
What does this mean? That you need a `nested_thing` and an `other_thing`, ie. an array of 2 elements?